base16-theme.el

```diff
@@ -822,6 +844,11 @@ return the actual color value.  Otherwise return the value unchanged."
      (sml/prefix                                   :inherit sml/global :foreground base09)
      (sml/read-only                                :inherit sml/not-modified :foreground base0C)
 
+;;;; solaire-mode
+     (solaire-default-face                         :inherit 'default :background (:darken base00 0.85))
+     (solaire-hl-line-face                         :inherit 'hl-line :background base00)
```

Comment:
I'm pretty sure the :inherit stuff doesn't need to be quoted

Reply:
Ok! Let me fix it. It's weird because themes across Emacs hold no consistent format so this kind of things happen

Reply:
Yeah, and most of my themes use a sort of custom theme "spec" to better support terminals and gui emacs better, making it even more confusing. Thanks for doing this!